modules/ossm-integrating-distr-tracing-tempo.adoc

```diff
@@ -0,0 +1,90 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-integrating-distr-tracing-tempo_{context}"]
+= Integrating distributed tracing platform (Tempo)
+//this one is specific to Tempo
+//another module may be specific to OTEL
+//another module may be specific to Kiali
+//they should all reside in ossm-observability, and in the order they need to happen/be completed
+
+You can expose tracing data to the {DTProductName} (Tempo) stack by appending a named element and `zipkin` provider to the `spec.meshConfig.extensionProviders` specification. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+
+.Prerequisites
+
+Install the {DTProductName} (Tempo) stack by following the instructions provided in 
+ifndef::openshift-rosa,openshift-dedicated[]
+xref:../../distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc[Installing the distributed tracing platform (Tempo)].
+endif::[]
+
+.Procedure
+
+. Configure the `ServiceMeshControlPlane` resource to connect to the Tempo Stack instance in the `tracing-system` namespace:
++
+[source,yaml]
+----
+kind: ServiceMeshControlPlane
+apiVersion: maistra.io/v2
+metadata:
+  name: basic
+  namespace: istio-system
+spec:
+...
+  meshConfig:
+    extensionProviders:
+      - name: tempo
+        zipkin:
+          service: tempo-sample-distributor.tracing-system.svc.cluster.local
+          port: 9411
+  tracing:
+    sampling: 10000
+    type: None <1>
+  version: v2.5
+---
+kind: ServiceMeshMemberRoll
+apiVersion: maistra.io/v1
+metadata:
+  name: default
+  namespace: istio-system
+spec:
+  members:
+    - tracing-system
+----
+<1> The `spec.tracing.type` setting defines a Distributed Tracing instance and 2 Jaeger instances. Set `spec.tracing.type` to `None` when connecting to a Tempo Stack using an `extensionProvider` setting.
+
+. Configure the Kiali resource specification to enable a Kiali workload Traces dashboard. You can use the dashboard to view tracing query results.
++
+[source,yaml]
+----
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+...
+spec:  
+  external_services:
+    tracing:
+      enabled: true
+      in_cluster_url: 'http://tempo-sample-query-frontend.tracing-system.svc.cluster.local:16686'
+      url: '[Tempo query frontend Route url]'
+----
++
+
+. Apply a Telemetry custom resource for {SMProductShortName} to start the Tempo provider setting.
```

Comment:
I believe that this would need an accompanying command to apply the CR. something like `oc apply -f` or something. 

Check the TempoStack vs. Tempo usage.